rust/module.rs

```diff
@@ -577,7 +577,8 @@ pub fn module(ts: TokenStream) -> TokenStream {
         ));
     }
 
-    let file = std::env::var("RUST_MODFILE").unwrap();
+    let file =
+        std::env::var("RUST_MODFILE").expect("Unable to fetch key for environmental variable");
```

Comment:
Let's give a bit more information by telling the user which variable was missing:

```suggestion
        std::env::var("RUST_MODFILE").expect("Unable to fetch RUST_MODFILE environmental variable");
```